### PR TITLE
[Article]: Service Mesh: Consul

### LINKED ISSUE #2497: [Resource] Article: Consul Service Mesh
Contributors and community members are encouraged to create articles under https://layer5.io/resources. A new article fitting the following description is requested to be written by a community member:

**URL**: /resources/service-mesh/what-is-consul-service-mesh
**Article Title:** Service Mesh: Consul
**Topic:** Create a page under /resources that generally describes Consul.
**Core points to convey in the article:** 
- Genesis
- Governance
- Architecture
- Special Features
- Excerpt of Functional, Non-Functional data from the service mesh landscape (/landscape)

_Articles under the /resources area are to be timeless and broadly helpful to visitors. As such, these articles carry no publish date or byline._

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by deepeshaburse**
**Description**
PR to add an article on Consul Connect and it's working.

This PR fixes #2497 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/resources/articles/consul-service-mesh/index.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Service Mesh: Consul
```
Comment (debo19) [inline src/collections/resources/articles/consul-service-mesh/index.mdx:2]:
```suggestion
title: "Service Mesh: Consul"
```

Comment (debo19) [reply src/collections/resources/articles/consul-service-mesh/index.mdx:2]: This was breaking the build.

Comment (deepeshaburse) [reply src/collections/resources/articles/consul-service-mesh/index.mdx:2]: Oh okay, I'll keep that in mind for next time too.

Comment (leecalcote) [reply src/collections/resources/articles/consul-service-mesh/index.mdx]: Speaking of next time... what do we have lined up? If nothing, let's get it lined up. 🙂

File: src/collections/resources/articles/consul-service-mesh/index.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        <p> Trust can be exploited and with the increasing number of services, there are higher chances of breach. The Consul service mesh control plane can be configured to enforce mutual TLS (mTLS), and will automatically generate and distribute the TLS certificates for every service in the mesh. The certificates are used for both service identity verification and communication encryption. </p> 
+      </li>
+      <li> Simplify Application Security with Intentions 
+        <p> Communication between services is secure within the mesh once the service sidecar proxies have been set up. To designate which services are permitted to communicate with one another, you might want to build a more granular set of policies. Consul Intentions are used to limit which services can make requests or create connections and define access control for services through Connect. We can manage intentions via the UI, CLI, or API. The proxy or a natively integrated application enforces intentions on inbound connections or requests. </p> 
+      </li>
+    </ol>
+  </p>
+  
+  <p> Compatibility of Consul Connect:
+  <br>
```
Comment (leecalcote) [inline src/collections/resources/articles/consul-service-mesh/index.mdx:82]:
```suggestion
  <br />
```

File: src/collections/resources/articles/consul-service-mesh/index.mdx (hunk 1 above)
Comment (deepeshaburse) [reply src/collections/resources/articles/consul-service-mesh/index.mdx]: Nothing is lined up as of now, but ready to get started with the next issue!

**Comment by leecalcote** [comment]
@deepeshaburse, uh-oh, bad file path reference - https://github.com/layer5io/layer5/runs/7696125084?check_suite_focus=true#step:3:86

**Comment by leecalcote** [comment]
Execute these checks to sign-off on your git commit(s) - https://github.com/layer5io/layer5/pull/3035/checks?check_run_id=7696124388

**Comment by l5io** [comment]
🚀 Preview for commit 30f0fcf43ed467d376cd939681a3c665187a811a at: https://62eda75647397957b0345f7a--layer5.netlify.app

**Comment by deepeshaburse** [comment]
> @deepeshaburse, uh-oh, bad file path reference - https://github.com/layer5io/layer5/runs/7696125084?check_suite_focus=true#step:3:86

I'm sorry I'm not sure how to it, can you help me with it?

**Comment by deepeshaburse** [comment]
> > @deepeshaburse, uh-oh, bad file path reference - https://github.com/layer5io/layer5/runs/7696125084?check_suite_focus=true#step:3:86
> 
> I'm sorry I'm not sure how to fix it, can you help me with it?

**Comment by l5io** [comment]
🚀 Preview for commit 57eb1adcecbee1749cf1b24dbbc12ea77a17851c at: https://62f007cd8b5f9f450f324efc--layer5.netlify.app

**Review by debo19 — APPROVED** [review]
LGTM 🎉 Thank you @deepeshaburse

**Comment by leecalcote** [comment]
Yay! Congratulations, @deepeshaburse! 🎈

**Comment by deepeshaburse** [comment]
Thank you @leecalcote and @debo19 for all your help 😄

**Comment by leecalcote** [comment]
@deepeshaburse, absolutely. 🙌 What's next on the plate?

**Pull request merged**